frontend/src/styles/tokens.scss

```diff
@@ -95,6 +95,7 @@ $color-violet-10: #f6b8ff;
 $color-violet-05: #f7e2ff;
 
 // Blue
+$color-blue-100: #0060df66;
```

Comment:
Question (non-blocking): Is there a reason one of the existing blues wasn't good? I wonder if its a good precedent to add the alpha channel in this hex code.

Reply:
Realized #0060df is $color-informational, changed this to rgba($color-informational, 0.4)